### PR TITLE
Feat/backend business logic

### PR DESCRIPTION
**Pull request by ilyasfoo**
Sorry this one is quite huge, cant find the time to split to chunks. Most crucial changes are in ` backend/src/loaders/lockdown` so that's hopefully a good filter to review.

Summary:
- Added business logic for producing "snapshots". For V1 it would only produce the "current date" snapshot.
- Added types for better refactoring in future (JSDocs  `typedef` seemed a bit weak, used classes instead)

Sample territory json:
```json
{
  "lockdown": {
    "date": "06 April 2020",
    "entry_timestamp": "05 April 2020",
    "type": "Media",
    "title_of_status": "State of Calamity",
    "max_gathering": 2,
    "measures": [{
      "label": "lockdown_status",
      "value": "1"
    }, {
      "label": "city_movement_restriction",
      "value": "1"
    }, {
      "label": "attending_religious_sites",
      "value": "3"
    }, {
      "label": "going_to_work",
      "value": "1"
    }, {
      "label": "military_not_deployed",
      "value": "4"
    }, {
      "label": "academia_allowed",
      "value": "3"
    }, {
      "label": "going_to_shops",
      "value": "2"
    }, {
      "label": "electricity_nominal",
      "value": "1"
    }, {
      "label": "water_nominal",
      "value": "1"
    }, {
      "label": "internet_nominal",
      "value": "1"
    }],
    "travel": {
      "land": [{
        "label": "local",
        "value": "1"
      }, {
        "label": "nationals_inbound",
        "value": "2"
      }, {
        "label": "nationals_outbound",
        "value": "2"
      }, {
        "label": "foreigners_inbound",
        "value": "2"
      }, {
        "label": "foreigners_outbound",
        "value": "2"
      }, {
        "label": "cross_border_workers",
        "value": "4"
      }, {
        "label": "commerce",
        "value": "1"
      }, {
        "label": "stopovers",
        "value": "5"
      }],
      "flight": [{
        "label": "local",
        "value": "1"
      }, {
        "label": "nationals_inbound",
        "value": "2"
      }, {
        "label": "nationals_outbound",
        "value": "2"
      }, {
        "label": "foreigners_inbound",
        "value": "2"
      }, {
        "label": "foreigners_outbound",
        "value": "5"
      }, {
        "label": "cross_border_workers",
        "value": "5"
      }, {
        "label": "commerce",
        "value": "5"
      }, {
        "label": "stopovers",
        "value": "5"
      }],
      "sea": [{
        "label": "local",
        "value": "5"
      }, {
        "label": "nationals_inbound",
        "value": "5"
      }, {
        "label": "nationals_outbound",
        "value": "5"
      }, {
        "label": "foreigners_inbound",
        "value": "5"
      }, {
        "label": "foreigners_outbound",
        "value": "5"
      }, {
        "label": "cross_border_workers",
        "value": "5"
      }, {
        "label": "commerce",
        "value": "5"
      }, {
        "label": "stopovers",
        "value": "5"
      }]
    }
  }
}
```

### CODE REVIEW

**Comment by thepassle** [comment]
@ilyasfoo Looks like you have some merge conflicts, could you take a look at those?

From what I can tell by looking at the output json files (worldmap/territories), the files look good 🙂

File: .github/workflows/update-data.yml [hunk 1]
```diff
@@ -31,12 +31,7 @@ jobs:
           git config user.email "lockdown-bot@non-existing.com"
           git config user.name "lockdown-bot"
           git add data
-          changed_files=$(git status --porcelain --untracked-files=no | wc -l)
-
-          if [ $changed_files -gt 0 ]
```
Comment (LarsDenBakker) [inline .github/workflows/update-data.yml:36]: looks like this one got mixed up in a merge conflict, this should be there

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Project Lockdown
+# Lockdown
```
Comment (LarsDenBakker) [inline README.md:1]: same here, some commits got reverted

**Comment by LarsDenBakker** [comment]
Something went wrong merging/rebasing on master, that's something to look into. Other than that I think it's fine to merge this, it's easier to review the business logic changes along with the front-end. Then we can do fixes afterwards.

**Comment by ilyasfoo** [comment]
Thanks for noticing, probably happened when I had wrongly used merge strategy when pulling in data conflicts.

I've re-committed all the changes, plus had to run `format` for frontend. Let me know if anything is still amiss.

**Comment by LarsDenBakker** [comment]
I think you're using an earlier version of the formatter, can you try reinstall the root node modules without a package lock? And then reformat?

**Comment by ilyasfoo** [comment]
@LarsDenBakker good catch, I've updated and removed the format commit

**Comment by LarsDenBakker** [comment]
Awesome!

**Pull request merged**